googletest/test/gtest_list_tests_unittest.py

```diff
@@ -71,7 +71,7 @@
 TypedTest/0\.  # TypeParam = (VeryLo{245}|class VeryLo{239})\.\.\.
   TestA
   TestB
-TypedTest/1\.  # TypeParam = int\s*\*
+TypedTest/1\.  # TypeParam = int\s*\*( __ptr64)?
```

Comment:
There seems to be a lot going on here that isn't technically necessary.
I would like to have some idea of what's being changed but I can't review all of this at once.
Can you write a little in the "conversation" tab about what exactly is being changed and why?